src/huggingface_inference_toolkit/webservice_starlette.py

```diff
@@ -69,6 +69,18 @@ async def health(request):
     return PlainTextResponse("Ok")
 
 
+# Report Prometheus metrics
+# inf_batch_current_size: Current number of requests being processed
+# inf_queue_size: Number of requests waiting in the queue
+async def metrics(request):
+    batch_current_size = MAX_CONCURRENT_THREADS - MAX_THREADS_GUARD.value
+    queue_size = MAX_THREADS_GUARD.statistics().tasks_waiting
+    return PlainTextResponse(
+        f"inf_batch_current_size {batch_current_size}\n" +
+        f"inf_queue_size {queue_size}\n"
```

Comment:
Not sure the naming is the best, I copied TGI

Reply:
Are we okay that this will always be 1?

Reply:
If it bother you or makes it unclear for customers I can remove `inf_batch_current_size`